integrations/nginx/nginx-monitoring/sensu-integration.yaml

```diff
@@ -21,6 +21,33 @@ spec:
     - "@nixwiz"
     - "@calebhailey"
   prompts:
+    - type: section
+      title: Configure NGINX Monitoring Thresholds
+    - type: markdown
+      body: |
+        This integration collects metrics from NGINX.
+        Configure the default alerting thresholds for active and waiting connections.
```

Comment:
```suggestion
        Configure the alerting thresholds for active and waiting connections.
```

Reply:
Since we're not using Sensu tokens, this isn't just a default - it's _the_ threshold